source/features/circular-movement-on-sidebar-lists.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * With this feature RGH will support cycled lists with keyboard support for arrow-up to navigate to the last item
+ * when pressed at the first one and for arrow-down to navigate to the first item when pressed at the last one
+ */
+import select from 'select-dom';
+import features from '../libs/features';
+
+function swapLabelFocus(labels: HTMLElement[], from: number, to: number): void {
+	labels[from].classList.remove('navigation-focus');
+	labels[from].setAttribute('aria-selected', 'false');
+	labels[to].classList.add('navigation-focus');
+	labels[to].setAttribute('aria-selected', 'true');
+}
+
+function performSwapLabelBehaviour(event: KeyboardEvent, labels: HTMLElement[], from: number, to: number): void {
+	event.stopImmediatePropagation();
+	event.preventDefault();
+	swapLabelFocus(labels, from, to);
+}
+
+function init(): void {
+	document.addEventListener('keydown', event => {
```

Comment:
This should use `delegate-it` like in my previous PR https://github.com/sindresorhus/refined-github/pull/1569/files (`onetime` is no longer needed though)